src/components/generic/button/Button.js

```diff
@@ -3,22 +3,30 @@ import PropTypes from 'prop-types';
 
 import './Button.scss';
 
-const Button = ({children, name, onClick, variant, color, enabled, id}) => {
+const Button = ({children, name, onClick, variant, color, enabled, id, ref}) => {
```

Comment:
We also could render a variant in this file like <LinkedButton /> which would render an <a> element with some property (would rename Button.js into Buttons.Js(x)

both could be exported and then imported like so : {LinkedButton (or Button)} from 'Buttons.js'

may be overkill for now as there's only one linked button 😄 so I didn't do it yet